src/Sql/Sql.Test/ScenarioTests/ServerKeyVaultKeyTests.cs

```diff
@@ -41,14 +41,14 @@ public ServerKeyVaultKeyTests(ITestOutputHelper output) : base(output)
             };
         }
 
-        [Fact] 
+        [Fact(Skip = "Requires hardcoded resource 'akvtdekeyvaultcl'")]
```

Comment:
We need to ge a positive confirmation from the test owner (or other knowledgeable dev) that this behavior has not regressed.